<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sgsApp</class>
 <widget class="QMainWindow" name="sgsApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Student Grading System</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidgetSGS">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>-3</y>
      <width>1451</width>
      <height>911</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="loginScreen">
     <property name="styleSheet">
      <string notr="true">background:url(:/images/background.jpg);
background-position:center;
background-repeat: no-repeat;</string>
     </property>
     <widget class="QFrame" name="loginBox">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>250</y>
        <width>291</width>
        <height>331</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #8a1738;
border-radius:10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="usernameInput">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff;
color: #333;
border:0px; 
border-top-right-radius:5px;
border-bottom-right-radius:5px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;</string>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordInput">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff;
color: #333;
border:0px; 
border-top-right-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signInButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>231</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fbaf01;
color: #ffffff;
border:0;
height:30px;
border-radius:5px;</string>
       </property>
       <property name="text">
        <string>SIGN IN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signUpButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px; color:#fff;</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/icons White/Icons/White/Add User.png</normaloff>:/icons White/Icons/White/Add User.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="forgotPasswordButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>280</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px;
color:#fff;
text-decoration:underline;</string>
       </property>
       <property name="text">
        <string>Forgot Password?</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff; font-size:28px; border:0px;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QFrame" name="usernameIcon">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff url(:/images/user.png);
border:0px;
border:0px;
border-top-left-radius:5px;
border-bottom-left-radius:5px;
border-top-right-radius:0px;
border-bottom-right-radius:0px; 
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="passwordIcon">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff url(:/images/lock.png);
border:0px;
border-radius:0px;
border-top-left-radius:5px;
border-bottom-left-radius:5px;
border-top-right-radius:0px;
border-bottom-right-radius:0px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="loginErrorLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px;
color:#b50009;
text-align:center;
background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="passwordShowButton">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#fff;
border:0px; 
border-top-right-radius:5px;
border-bottom-right-radius:5px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/iconsGray/Icons/Gray/Eye ON.png</normaloff>:/iconsGray/Icons/Gray/Eye ON.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="footerLogo">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>180</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>61</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/images/sideBarLogo.png);
background-position:center;
background-repeat: no-repeat;
border: 0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>462</width>
        <height>281</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>462</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/SGS LOGO.png);
border:0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Dashboard">
     <property name="minimumSize">
      <size>
       <width>1440</width>
       <height>900</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1440</width>
       <height>900</height>
      </size>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>-10</y>
        <width>2</width>
        <height>2</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
     <widget class="QFrame" name="Sidebar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>260</width>
        <height>900</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>900</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>900</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Add</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#292f38;
border: 0;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>730</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:url(:/images/sideBarLogo.png);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>191</width>
         <height>641</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#70808c;</string>
       </property>
       <property name="title">
        <string>Main Menu</string>
       </property>
       <widget class="QPushButton" name="dashboard_pushButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>86</width>
          <height>16</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#fff;</string>
        </property>
        <property name="text">
         <string>Dashboard</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons White/Icons/White/Dashboard.png</normaloff>:/icons White/Icons/White/Dashboard.png</iconset>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>161</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>Reports</string>
        </property>
        <widget class="QPushButton" name="viewComments_Button">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>41</y>
           <width>86</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Add Institution</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>Comments</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Eye ON.png</normaloff>:/icons White/Icons/White/Eye ON.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="viewCoursesPending_Button">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>71</y>
           <width>118</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Add Institution</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>Pending Course</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Eye ON.png</normaloff>:/icons White/Icons/White/Eye ON.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="viewCourses_Button">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>101</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Add Institution</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>Courses</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Eye ON.png</normaloff>:/icons White/Icons/White/Eye ON.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="viewProgramSequence_Button">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Add Institution</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>Program Sequence</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Eye ON.png</normaloff>:/icons White/Icons/White/Eye ON.png</iconset>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="settingGroupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>340</y>
          <width>160</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <widget class="QPushButton" name="addInstitution_Button">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>41</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Add Institution</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>Institution</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Add.png</normaloff>:/icons White/Icons/White/Add.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="addCourse_Button">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>71</y>
           <width>64</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Add Cours</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>Course</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Add.png</normaloff>:/icons White/Icons/White/Add.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="AddProgramSequence_Button">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>101</y>
           <width>136</width>
           <height>16</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>Program Sequence</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Add.png</normaloff>:/icons White/Icons/White/Add.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="changePassword_Button">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>131</y>
           <width>130</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Add Institution</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>Change Password</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Edit.png</normaloff>:/icons White/Icons/White/Edit.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="addUser_Button">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>161</y>
           <width>102</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Add Institution</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>User Account</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Add User.png</normaloff>:/icons White/Icons/White/Add User.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="disableUser_Button">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>191</y>
           <width>102</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Add Institution</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#fff;</string>
         </property>
         <property name="text">
          <string>User Account</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons White/Icons/White/Eye OFF.png</normaloff>:/icons White/Icons/White/Eye OFF.png</iconset>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="headBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1440</width>
        <height>65</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1440</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/background.jpg);
border:0;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="logoutButton">
       <property name="geometry">
        <rect>
         <x>1290</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string> Logout</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/icons White/Icons/White/Passsword.png</normaloff>:/icons White/Icons/White/Passsword.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="userRoleLable_2">
       <property name="geometry">
        <rect>
         <x>1280</x>
         <y>20</y>
         <width>16</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>|</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="userIconTopBar">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/icons White/Icons/White/User.png</normaloff>:/icons White/Icons/White/User.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="userRoleLable">
       <property name="geometry">
        <rect>
         <x>1230</x>
         <y>28</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>, STUDENT</string>
       </property>
      </widget>
      <widget class="QFrame" name="topBarLogo">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/SGS Logo Dashboard.png);
background-repeat: no-repeat;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="usernameLable">
       <property name="geometry">
        <rect>
         <x>1160</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Federico</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
